src/main/java/com/google/sps/servlets/NameServlet.java

```diff
@@ -0,0 +1,50 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.sps.servlets;
+
+import com.google.gson.Gson;
+import java.io.IOException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+@WebServlet("/name-data")
+public class NameServlet extends HttpServlet {
+  private static final String NAME_PARAMETER = "name-input";
+  private static final String JSON_TYPE = "application/json";
```

Comment:
Since this constant (and probably others eventually) will be used in multiple files, could you create a Constants.java (or ResponseTypes.java etc. if you want to divide them up more, but I think that just sectioning them within a general Constants file is good enough) for holding these shared constants? OK to leave this as a TODO comment rather than changing in this PR.